sdk/src/feature_set.rs

```diff
@@ -687,6 +687,10 @@ pub mod reduce_stake_warmup_cooldown {
     }
 }
 
+pub mod feature_gate_program {
```

Comment:
"verb" + noun is a bit more descriptive
```suggestion
pub mod programify_feature_gate_program {
```